examples/mysql-atomic-transaction/mysql_atomic_transaction.md

```diff
@@ -1,13 +1,13 @@
 # Database Access - Atomic transactions
 
-This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations with the help of a `transaction` to achieve the atomic behaviour.
+The `mysql` client supports atomic units of work with multiple SQL statements. You can use the Ballerina `transaction` package with the `mysql` client to achieve atomic database transactions. The database makes all changes permanent when the transaction is committed or undo all changes when the transaction is rolled back.
```

Comment:
The `mysql:Client` supports atomic units of work with multiple SQL statements. To achieve atomic database transactions use the Ballerina `transaction` package with the `mysql:Client` . The database makes all changes permanent when the transaction is committed or undo all changes when the transaction is rolled back.

Reply:
Fixed in [6210bb5](https://github.com/ballerina-platform/ballerina-distribution/pull/3770/commits/6210bb5561661064f55651540c3f5c82a6610333)